pkg/pipelines/pipelines.go

```diff
@@ -261,3 +261,26 @@ func AssertNumberOfTaskruns(c *clients.Clients, namespace, numberOfTr, timeoutSe
 		assert.FailOnError(fmt.Errorf("Error: Expected %v number of taskruns but found %v number of taskruns", numberOfTr, len(trlist.Items)))
 	}
 }
+
+func AssertPipelinesExist(c *clients.Clients, status, namespace string) {
```

Comment:
Consider splitting it into two - `AssertPipelinesExist` and `AssertPipelinesDontExist` so that the code is more readable.

Now we call it like `pipelines.AssertPipelinesExist(c, "not present", "openshift")`, with proposed change it would become `pipelines.AssertPipelinesDontExist(c, "openshift")` (the if condition would be extracted to the file `steps/pipeline/pipeline.go`

Reply:
I thought of implementing it in the same way, but I thought of creating a generalized function, and as you mentioned it is not readable. I will update my PR